README.md

```diff
@@ -1,4 +1,4 @@
-# Viewing Party
+# Viewing Party - Ansel (Snow Leopards)
```

Comment:
Comments can be left on individual lines or groups of lines.

There are two main ways that code suggestions are made:

1. Code quoted suggestion
```py
    title = "Viewing Party - Ansel (Snow Leopards)"
```

2. Suggestion quoting
```suggestion
    title = "Viewing Party - Ansel (Snow Leopards)"
```